### ISSUE TITLE
User Agent

### ISSUE THREAD

**Issue by alfg** [bartok765/galaxy_blizzard_plugin#34]
Hi there,

I'm the maintainer of https://github.com/alfg/overwatch-api, and I noticed you use https://ow-api.io as an integration for your project (Thank you!).

I am curious why you are using the user agent defined at:
https://github.com/bartok765/galaxy_blizzard_plugin/blob/67acdeda76f5b6681939d28004d1e30fc49797c0/src/consts.py#L30

And if it's possible to use a custom user-agent as part of your HTTP client requests. I am not familiar with the GOG Galaxy client and it's plugin ecosystem, so I may be missing some context.

Thank you.

**Comment by bartok765** [comment]
Hi @alfg 

This repo follows @FriendsOfGalaxy work so to be honest no idea, why Firefox agent was used originally.

My guess is the whole auth/refresh flow is imitating a bit complicated browser authentication flow (not what public blizzard API exposes). Maybe it is related?

Why would you like to change it? I guess for https://ow-api.io metrics?

**Comment by alfg** [comment]
Right. I just noticed a good amount of traffic with that user agent and was curious. But I understand if it's for a complicated auth flow if it cannot be updated.
